runtime/helpers.js

```diff
@@ -22,6 +22,26 @@ function notEmpty(o) {
     return true;
 }
 
+function createLazyRenderer(handler) {
```

Comment:
Is it really a lazyRenderer ? seem like the handler.renderer is abstracted here ..